### PR TITLE
Update developer.md

### PR DESCRIPTION
**Pull request by benjaminmisell**
Update to use go get instead of git clone in dev docs.

### CODE REVIEW

File: docs/developer.md [hunk 1]
```diff
@@ -11,22 +11,14 @@ your repository with the upstream openebs/maya master. Syncing ensures your repo
...
-```sh
-working_dir=$GOPATH/src/github.com/openebs
-```
 Create your clone:
 
 ```sh
-mkdir -p $working_dir
-cd $working_dir
-git clone https://github.com/$user/maya.git 
+go get github.com/$user/maya.git
```
Comment (kmova) [inline docs/developer.md]: @benjaminmisell -- Thanks for looking into this! 

My first thought, i might be a bit outdated here.. does go support compiling this?  doesn't it expect that the forked repository (github.com/$user/maya) be actually under ( github.com/openebs/maya)

Comment (benjaminmisell) [reply docs/developer.md]: I belive it will download the fork and summing it to the original as well to keep compatibility.

Comment (kmova) [reply docs/developer.md]:
```
kiran@kmaya:~/go/src/github.com/openebs$ go get github.com/kmova/maya.git
package github.com/kmova/maya.git: invalid version control suffix in github.com/ path
```

I also tried with *go get github.com/kmova/maya*, this downloaded the code into kmova/maya. The make is returning the following error:

```
kiran@kmaya:~/go/src/github.com/kmova/maya$ make apiserver
GOOS=linux go build ./cmd/apiserver
cmd/apiserver/commands.go:7:2: cannot find package "github.com/openebs/maya/cmd/apiserver/app/command" in any of:
	/home/kiran/go/src/github.com/kmova/maya/vendor/github.com/openebs/maya/cmd/apiserver/app/command (vendor tree)
	/usr/local/go/src/github.com/openebs/maya/cmd/apiserver/app/command (from $GOROOT)
	/home/kiran/go/src/github.com/openebs/maya/cmd/apiserver/app/command (from $GOPATH)
GNUmakefile:103: recipe for target 'apiserver' failed
make: *** [apiserver] Error 1
```

Comment (benjaminmisell) [reply docs/developer.md]: Oh sorry my bad. You need to go get the original then cd into it and set the origin to the fork. Will ammend my pr when I have time

Comment (kmova) [reply docs/developer.md]: Thank you @benjaminmisell

File: docs/developer.md [hunk 2]
```diff
@@ -11,25 +11,21 @@ your repository with the upstream openebs/maya master. Syncing ensures your repo
...
 # Note: Here user= your github profile name
+git clone https://github.com/$user/maya.git
 
-cd $working_dir/maya
+# Configure remote upstream
+cd $GOPATH/github.com/openebs/maya
 git remote add upstream https://github.com/openebs/maya.git
 
-# Never push to upstream master
+# Never push to upstream mastr
```
Comment (prateekpandey14) [inline docs/developer.md]: master

**Pull request merged**